tso-server/main.go

```diff
@@ -13,6 +14,11 @@ import (
 	"github.com/ngaut/tso/proto"
 )
 
+var (
+	addr     = flag.String("addr", ":1234", "server listening address")
+	netProto = flag.String("proto", "tcp", "server network protocol")
```

Comment:
Since we don't support other protocols, there is no need to support protocol flag.